synapse/lib/gis.py

```diff
@@ -0,0 +1,107 @@
+import math
+
+import synapse.lib.syntax as s_syntax
+
+'''
+Synapse module with helpers for geo-spacial calculations.
+'''
+
+# base geo distances will be in mm
+r_mm = 6371008800.0
+r_km = 6371.0088
+
+# investigate perf impact of using WGS-84 ellipsoid for dist calc
+
+def latlong(text):
```

Comment:
The `latlong` function here seems redundant w/ `LatLongType`'s norm in synapse/models/geospace.py but does not validate the data.

Reply:
not redundant because the norm routine still returns a string.  meant for simple/fast chop of already normalized latlong strings.